package.json

```diff
@@ -67,7 +67,7 @@
     "resolve": "1.12.0",
     "semver": "6.3.0",
     "string-width": "3.1.0",
-    "typescript": "3.6.4",
+    "typescript": "3.8.0-dev.20191024",
```

Comment:
Please switch to `3.7.1-rc`. `3.8.0-dev` can (and seemingly does) contain new breaking changes.

Reply:
oh, accidentally upgraded to 3.8.0